README.md

```diff
@@ -39,15 +39,15 @@ This builds the plugin in `/build/rootfs/`
 ### Run
 Run the snap agent with the config file
 
-> $GOPATH/bin/snap-v0.14.0-beta/bin/snapd --plugin-trust 0 --log-level 1 --config $GOPATH/src/github.com/intelsdi-x/snap-plugin-collector-nginx/config.json
```

Comment:
I replaced $GOPATH with $SNAP_PATH and removed reference to exact version completely, is it right?